contracts/contracts/token/OUSD.sol

```diff
@@ -143,23 +155,23 @@ contract OUSD is Initializable, InitializableERC20Detailed, Governable {
             // For a period before the resolution upgrade, we created all new
             // contract accounts at high resolution. Since they are not changing
             // as a result of this upgrade, we will return their true values
-            return (_creditBalances[_account], cpt);
+            return (creditBalances[_account], cpt);
         } else {
```

Comment:
This else block can be removed.

Reply:
Ah - I see what you mean. The code inside it could come out of the block and live at the top level.

Probably won't change at this point since it's just been audited as is, and it's a fairly close stylistic choice.